### PR TITLE
cmdport

### PR DESCRIPTION
**Pull request by przemas75**
https://github.com/aboe76/puppet-chrony/issues/28

### CODE REVIEW

**Comment by canihavethisone** [comment]
I can see an issue (1) and have a recommendation (2) with these changes.
1. cmdport defaults to 323 but you have defaulted it to 257
2. as it defaults to 323 if not defined in chronyd.conf, make it undef in params.pp and use an if statement in the templates so it only gets written if defined

File: manifests/init.pp [hunk 1]
```diff
@@ -2,6 +2,7 @@
 class chrony (
   Array[String] $bindcmdaddress     = $chrony::params::bindcmdaddress,
   Array[String] $cmdacl             = $chrony::params::cmdacl,
+  $cmdport                    = $chrony::params::cmdport,
```
Comment (aboe76) [inline manifests/init.pp]: can you fix the allignment please?

**Comment by aboe76** [comment]
Merged it

**Comment by aboe76** [comment]
Thanks for the hard work

**Pull request merged**